tests/manual-tests/framework/preferences/fullPage/index.html

```diff
@@ -87,6 +87,7 @@
                     </li>
                     <li>Use the "save and apply" button to apply the preview styles to Preferences Editor</li>
                     <li>Use the "reset" button to reset the styles</li>
+                    <li>Be sure to test with the <a href="index-jQueryUI.html">jQuery UI widgets version as well.</a></li>
```

Comment:
I think that it's good to have the link here, but I'm also thinking that it might be hard to discover the alternate versions of these tests because of the use of index.html and the fact that testers may not read the instructions. I wonder if separating the versions out into separate sub directories would help with this and keeping the links the instructions as well.